drod/PuzzleModeOptions.h

```diff
@@ -0,0 +1,103 @@
+// $Id$
+
+/* ***** BEGIN LICENSE BLOCK *****
+ * Version: MPL 1.1
+ *
+ * The contents of this file are subject to the Mozilla Public License Version
+ * 1.1 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * http://www.mozilla.org/MPL/
+ *
+ * Software distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License
+ * for the specific language governing rights and limitations under the
+ * License.
+ *
+ * The Original Code is Deadly Rooms of Death.
+ *
+ * The Initial Developer of the Original Code is
+ * Caravel Software.
+ * Portions created by the Initial Developer are Copyright (C) 2001, 2002, 2005, 2020
+ * Caravel Software. All Rights Reserved.
+ *
+ * Contributor(s):
+ * Maurycy Zarzycki (maurycy@evidentlycube.com)
+ *
+ * ***** END LICENSE BLOCK ***** */
+
+#ifndef PUZZLEMODEOPTIONS_H
+#define PUZZLEMODEOPTIONS_H
+
+#include <BackEndLib/Types.h>
+#include <DRODLib/DbPackedVars.h>
+#include <DRODLib/SettingsKeys.h>
+
+struct PuzzleModeOptions {
+	PuzzleModeOptions() {
```

Comment:
For the default constructor, lets use the standard construction variable initialization idiom, i.e.,
`Constructor() : var1(val), var2(val), ... { }`